zulip_bots/setup.py

```diff
@@ -66,6 +66,7 @@
         "lxml",
         "BeautifulSoup4",
         "typing_extensions",
+        "importlib-metadata >= 3.6",
```

Comment:
Should have an [environment marker](https://www.python.org/dev/peps/pep-0496/) `python_version < '3.10'` or something.